commands/assign.py

```diff
@@ -57,7 +57,7 @@ def assign(conf, args):
                     repo = StudentRepo.new(base, semester, student_section,
                                            username, token)
                     repo.push(base, branch)
-                    repo.unprotect(branch)
+                    repo.protect(branch)
```

Comment:
Sanity check: don't new repos start with these settings? (`master` is protected, developers can push, and developers can merge)